bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -456,13 +456,12 @@ sitemap openwebnet label="OpenWebNet Binding Example Sitemap"
     
     Frame label="Alarm"
     {
-         Switch  item=iAlarm_System_State       label="Alarm state"
-         Switch  item=iAlarm_Control            label="Arm/Disarm alarm"   icon="shield"
-         Switch  item=iAlarm_System_Armed       label="Armed"              icon="shield"
-         Switch  item=iAlarm_System_Network     label="Network"            icon="network"
-         Default item=iAlarm_System_Battery     label="Battery"            icon="battery"
-         Switch  item=iAlarm_Zone_3_State       label="Zone 3 state"
-         Default item=iAlarm_Zone_3_Alarm       label="Zone 3 alarm"       icon="siren"
+         Selection item=iAlarm_Control            label="Arm/Disarm alarm"   icon="shield" mappings=[OFF="Disarm", ON="Arm"] 
```

Comment:
You can keep a Switch element, of course keeping the mappings you added. This will show two buttons, "Disarm" and "Arm". That is very probably better than a selection in this case.

Reply:
Ok I will try as  you suggested

Reply:
Ok it works